synapse/tests/test_cortex.py

```diff
@@ -152,7 +151,7 @@ async def test_cortex_edges(self):
 
             # we should now be able to edge walk *and* refs in
             nodes = await core.nodes('inet:ipv4=1.2.3.4 <-- *')
-            #self.len(2, nodes)
+            # self.len(2, nodes)
```

Comment:
if we're going to bother editing this, lets just remove it.

Reply:
ok